webui/react/src/App.tsx

```diff
@@ -171,16 +133,18 @@ const AppView: React.FC = () => {
             <>
               {isServerReachable ? (
                 <ConfirmationProvider>
+                  {/* Global app components: */}
                   <ClusterMessageBanner message={info.clusterMessage} />
+                  <JupyterLabGlobal
+                    enabled={
+                      Loadable.isLoaded(loadableUser) &&
+                      (workspace ? canCreateWorkspaceNSC({ workspace }) : canCreateNSC)
+                    }
+                    workspace={workspace ?? undefined}
+                  />
+                  <Omnibar />
+                  <VersionChecker version={info.version} />
```

Comment:
its not from this PR, but `info.version` returns something like this on latest-main. looks like `version !== process.env.VERSION` always return true? 

<img width="418" alt="image" src="https://github.com/user-attachments/assets/b625af2f-95e5-4d4b-9a28-87aeb27e1fb5">

Reply:
showing commit hash looks new but I think it applies both to process.env.VERSION and version returned by `/master` API call so I don't think it would always return true